### PR TITLE
Suspension Management project

### PR DESCRIPTION
**Pull request by miquelgall**

### CODE REVIEW

File: samples/003-SuspensionMng/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# getUsers
```
Comment (btoron) [inline samples/003-SuspensionMng/README.md:1]: Change name to SuspensionMgr

File: samples/003-SuspensionMng/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+None
+
+**Open Parameters**
+
+- `openAction`: Action that indicates if we are suspending or completing the activity. Possible values : [`suspend`, `complete suspended`]
+
+**Properties needed**
+
+None
```
Comment (btoron) [inline samples/003-SuspensionMng/README.md:20]: Can you please double check this? I thought the values were using some custom properties

File: samples/003-SuspensionMng/assets/js/utils/jsonview.ts [hunk 3]
```diff
@@ -0,0 +1,260 @@
+import "../../css/jsonviewer.scss";
+
```
Comment (btoron) [inline samples/003-SuspensionMng/assets/js/utils/jsonview.ts:2]: Check if this file is needed

File: samples/003-SuspensionMng/assets/css/jsonviewer.scss
Comment (btoron) [inline samples/003-SuspensionMng/assets/css/jsonviewer.scss:1]: Check if this file is needed

File: samples/.DS_Store
Comment (btoron) [inline samples/.DS_Store:1]: Remove

File: samples/003-SuspensionMng/plugin.xml
Comment (btoron) [inline samples/003-SuspensionMng/plugin.xml:1]: Remove

File: samples/003-SuspensionMng/README.md (hunk 2 above)
Comment (btoron) [reply samples/003-SuspensionMng/README.md:20]: According to the descriptor:
           "appt_number",
           "length",
           "ETA",
           "astatus",
           "aid",
           "date",
           "A_TOTAL_DURATION",
           "A_TOTAL_DURATION_COMPLETE",
           "A_ORIGINAL_START_TIME",
           "A_ORIGINAL_START_TIME_TZ",
           "A_TOTAL_TIME_TO_SOLVE"

**Review by btoron — CHANGES_REQUESTED** [review]

**Review by miquelgall — COMMENTED** [review]
Changes applied according to recommendations

File: samples/003-SuspensionMng/assets/js/utils/jsonview.ts (hunk 3 above)
Comment (miquelgall) [reply samples/003-SuspensionMng/assets/js/utils/jsonview.ts]: File removed

**Review by btoron — APPROVED** [review]

**Pull request merged**